cmd/promote.go

```diff
@@ -17,11 +17,20 @@ import (
 	"github.com/elastic/elastic-package/internal/promote"
 )
 
+const promoteLongDescription = `Use promote command to move packages between stages in package-storage.
+
+This command is intended primarily for use by administrators.
+
+It allows for selecting candidates for promotion among packages and opens new pull requests to review changes. Please be aware that the tool checks out an in-memory Git repository and switches over branches (snapshot, staging and production), so it may take longer to promote a larger number of packages.
```

Comment:
```suggestion
It allows for selecting packages for promotion and opens new pull requests to review changes. Please be aware that the tool checks out an in-memory Git repository and switches over branches (snapshot, staging and production), so it may take longer to promote a larger number of packages.
```